examples/server/rs-server.cpp

```diff
@@ -0,0 +1,134 @@
+// to initialize ours if we want to use the APIs!
+INITIALIZE_EASYLOGGINGPP
+#endif
+
+#define DBG CLOG(DEBUG,   "librealsense")
+#define ERR CLOG(ERROR,   "librealsense")
+#define WRN CLOG(WARNING, "librealsense")
+#define INF CLOG(INFO,    "librealsense")
+
+// Signal handler
+namespace {
+    std::function<void(int)> shutdown_handler;
+    void signal_handler(int signal) { shutdown_handler(signal); }
+}
+
+int main(int argc, char** argv)
+{
+    // Log engine initialization and configuration
+#ifdef BUILD_SHARED_LIBS
+    // Configure the logger
+    el::Configurations conf;
+    conf.set(el::Level::Global, el::ConfigurationType::Format, "[%level] %msg");
+    conf.set(el::Level::Global, el::ConfigurationType::ToStandardOutput, "true");
+
+    conf.set(el::Level::Trace,   el::ConfigurationType::Enabled, "false");
+    conf.set(el::Level::Debug,   el::ConfigurationType::Enabled, "false");
+    conf.set(el::Level::Fatal,   el::ConfigurationType::Enabled, "true");
+    conf.set(el::Level::Error,   el::ConfigurationType::Enabled, "true");
+    conf.set(el::Level::Warning, el::ConfigurationType::Enabled, "true");
+    conf.set(el::Level::Verbose, el::ConfigurationType::Enabled, "true");
+    conf.set(el::Level::Info,    el::ConfigurationType::Enabled, "true");
+    conf.set(el::Level::Unknown, el::ConfigurationType::Enabled, "true");
+
+    el::Loggers::reconfigureLogger("librealsense", conf);
+#else
+    rs2::log_to_callback(RS2_LOG_SEVERITY_INFO,
+        [&](rs2_log_severity severity, rs2::log_message const& msg)
+    {
+        std::cout << msg.raw() << "\n";
+    });
+#endif
+
+    INF << "Rs-server is running\n";
+
+    // Handle the command-line parameters
+    TCLAP::CmdLine cmd("LRS Network Extentions Server", ' ', RS2_API_VERSION_STR);
+    TCLAP::SwitchArg arg_enable_compression("c", "enable-compression", "Enable video compression");
+    TCLAP::ValueArg<std::string> arg_address("i", "interface-address", "Address of the interface to bind on", false, "", "string");
+    TCLAP::ValueArg<unsigned int> arg_port("p", "port", "RTSP port to listen on", false, 8554, "integer");
+    TCLAP::ValueArg<std::string> arg_serial("s", "serial-number", "Serial number of the camera", false, "", "string");
+
+    cmd.add(arg_enable_compression);
+    cmd.add(arg_address);
+    cmd.add(arg_port);
+    cmd.add(arg_serial);
+
+    cmd.parse(argc, argv);
+
+    std::string addr = "0.0.0.0";
+    if (arg_address.isSet())
+    {
+        addr = arg_address.getValue();
+    }
+
+    int port = 8554;
+    if (arg_port.isSet())
+    {
+        port = arg_port.getValue();
+    }
+
+    // Get the device
+    rs2::context ctx;
+    rs2::device dev;
+    bool dev_found = false;
+
+    std::string serial = "";
+    if (arg_serial.isSet())
+    {
+        serial = arg_serial.getValue();
+        for (auto&& device : ctx.query_devices()) {
+            std::string dev_serial = dev.get_info(RS2_CAMERA_INFO_SERIAL_NUMBER);
```

Comment:
Add a guard  - this info does not apply for platform camera

Reply:
Can you clarify?